Consul.Test/EventTest.cs

```diff
@@ -45,21 +45,15 @@ public void Dispose()
         [Fact]
         public async Task Event_FireList()
         {
-            var userevent = new UserEvent()
-            {
-                Name = "foo"
-            };
-
-            var res = await _client.Event.Fire(userevent);
-
-            await Task.Delay(100);
```

Comment:
We don't need any delay on client side because we use `WaitIndex` (https://www.consul.io/api/features/blocking).